lib/cdo/script_constants.rb

```diff
@@ -141,7 +143,11 @@ def self.position_in_category(script, category)
   end
 
   def self.category_priority(category)
-    CATEGORY_ORDERING_PRIORITY[category.to_sym] || 0
+    if category == OTHER_CATEGORY_NAME
+      1e6 # 'other' goes at the end
+    else
+      CATEGORIES.keys.find_index(category.to_sym)
+    end
```

Comment:
I do think this approach in general makes ordering much easier to reason about :)